gempy/library/nddops.py

```diff
@@ -283,47 +262,55 @@ def _num_good(mask):
         return np.sum(mask == False, axis=0)
 
     @stack_nddata
-    def __call__(self, data, mask=None, variance=None):
+    def __call__(self, data, mask=None, variance=None,
+                 save_rejection_map=False):
         """
         Perform the rejection and combining. The stack_nddata decorator
         allows a series of NDData object to be sent, and split into data, mask,
         and variance.
         """
-        rej_args = {arg: self._dict[arg] for arg in self._rejector.required_args
-                    if arg in self._dict}
 
         # Convert the debugging pixel to (x,y) coords and bounds check
         if self._debug_pixel is not None:
             try:
-                self._debug_pixel = np.unravel_index(self._debug_pixel, data.shape[1:])
+                self._debug_pixel = np.unravel_index(self._debug_pixel,
+                                                     data.shape[1:])
             except ValueError:
                 self._logmsg("Debug pixel out of range")
                 self._debug_pixel = None
-            else:
-                self._logmsg("Debug pixel coords {}".format(self._debug_pixel))
-                self._pixel_debugger(data, mask, variance, stage='at input')
-                self._logmsg("Rejection: {} {}".format(self._rejector.__name__, rej_args))
-
-        data, mask, variance = self._rejector(data, mask, variance, **rej_args)
-        #try:
-        #    data, mask, variance = self._rejector(data, mask, variance, **rej_args)
-        #except Exception as e:
-        #    self._logmsg(str(e), level='warning')
-        #    self._logmsg("Continuing without pixel rejection")
-        #    self._rejector = self.none
-        comb_args = {arg: self._dict[arg] for arg in self._combiner.required_args
-                     if arg in self._dict}
 
         if self._debug_pixel is not None:
-            self._pixel_debugger(data, mask, variance, stage='after rejection')
-            self._logmsg("Combining: {} {}".format(self._combiner.__name__, comb_args))
+            self._logmsg("Debug pixel coords {}".format(self._debug_pixel))
+            self._pixel_debugger(data, mask, variance, stage='at input')
+            info = data[(slice(None),) + self._debug_pixel]
+            self._logmsg("stats: mean={:.4f}, median={:.4f}, std={:.4f}"
+                         .format(np.mean(info), np.median(info), np.std(info)))
```

Comment:
Added a few info here, the mean/median/std at the beginning of the clipping, which can help to understand the clipping process.